lib/block.ml

```diff
@@ -42,7 +44,7 @@ end
 
 type section = int * string
 
-type cram_value = { non_det : Label.non_det option }
+type cram_value = { header : Header.t option; non_det : Label.non_det option }
```

Comment:
This type can be more precise, as only `Sh` and `Bash` variants are used

Reply:
True; done.